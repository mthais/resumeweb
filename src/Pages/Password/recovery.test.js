import React from 'react'
import { mount } from 'enzyme'
import RecoveryPasswordPage from './recovery'

describe('RecoveryPasswordPage', () => {
  let wrapper

  beforeEach(() => {
    wrapper = mount(<RecoveryPasswordPage />)
  })

  it('should pass with snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
