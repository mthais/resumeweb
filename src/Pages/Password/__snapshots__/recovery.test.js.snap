// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecoveryPasswordPage should pass with snapshot 1`] = `"<div class=\\"col\\"><h2 class=\\"text-center page-title\\">Esqueceu sua senha?</h2><p class=\\"text-wrapper\\">Digite seu endereço de e-mail abaixo e lhe enviaremos um link para redifinir sua senha.</p><form><div class=\\"position-relative form-group\\"><label for=\\"email\\" class=\\"mb-0\\">Email</label><div><input name=\\"email\\" id=\\"email\\" placeholder=\\"email@contato.com\\" type=\\"email\\" class=\\"form-control form-control\\" value=\\"\\"><div class=\\"invalid-feedback\\"></div></div></div><button class=\\"px-4 btn btn-primary\\">Enviar e-mail de redefinição de senha</button></form></div>"`;
